indra/llimage/llimagetga.cpp

```diff
@@ -1051,7 +1052,7 @@ bool LLImageTGA::decodeTruecolorRle8( LLImageRaw* raw_image )
 // Processing happens during the decode for speed.
 bool LLImageTGA::decodeAndProcess( LLImageRaw* raw_image, F32 domain, F32 weight )
 {
-	llassert_always(raw_image);
+	llassert_return_false(raw_image);
```

Comment:
I think this should stay either llassert_always or better be 'if (!raw_image) LL_ERRS', because if this happened, viewer encountered issues with decoder that are bigger than a single image failure.